universal-application-tool-0.0.1/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -1,31 +1,44 @@
 package controllers.applicant;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static play.inject.Bindings.bind;
 import static play.mvc.Http.Status.FOUND;
 import static play.mvc.Http.Status.OK;
 import static play.test.Helpers.contentAsString;
+import static play.test.Helpers.fakeRequest;
+import static play.test.Helpers.stubMessagesApi;
 
+import java.util.Collections;
+import java.util.Map;
 import models.Applicant;
 import models.Program;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
```

Comment:
```suggestion
import com.google.common.collect.ImmutableMap;
```

Reply:
Yikes, completely missed that - it keeps importing the wrong one. Thanks for catching!